clippy_dev/src/lib.rs

```diff
@@ -84,6 +85,23 @@ pub fn gen_changelog_lint_list(lints: Vec<Lint>) -> Vec<String> {
         .collect()
 }
 
+/// Generates the 'register_removed' code in `./clippy_lints/src/lib.rs`.
+pub fn gen_deprecated(lints: Vec<Lint>) -> Vec<String> {
+    lints.iter()
+        .filter(|l| l.deprecation.is_some())
+        .map(|l| {
+            format!(
+                r#"    store.register_removed(
+        "{}",
+        "{}",
+    );"#,
```

Comment:
This multi-line literal feels... brittle?

I found [indoc](https://github.com/dtolnay/indoc) by @dtolnay which could make this a little bit nicer. However, the code is unit tested and I'm not sure if it's worth adding a new dependency.

Reply:
oh, yeah. I forgot to run the dogfood tests locally :(

Reply:
It's not gonna catch that because pedantic lints are not (yet..) enabled I think.

Reply:
> pedantic lints are not (yet..) enabled I think.

Yep https://github.com/rust-lang-nursery/rust-clippy/issues/3172#issuecomment-432608735